docs/portal/image-pull-secrets/managing-image-pull-secrets.md

```diff
@@ -19,7 +19,7 @@ The following figure shows the key elements on the page, and the table following
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
- 1                         | To filter ImagePullSecrets, type characters in this field and then press the Enter key. All ImagePullSecrets that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                                                   |
-| 2                        | Each ImagePullSecret appears on its own row. Clicking an ImagePullSecret allows you to view basic information about the ImagePullSecret.                                                           |
-| 3                        | The Actions drop-down list on each row has options to [update](</docs/portal/image-pull-secrets/updating-image-pull-secret.md>) or [delete](</docs/portal/image-pull-secrets/deleting-an-image-pull-secret.md>) an ImagePullSecret.                                                           |
-| 4                        | The **+ Add New ImagePullSecret** button allows you to [add](</docs/portal/image-pull-secrets/adding-an-image-pull-secret.md>) ImagePullSecrets.                                                     |
\ No newline at end of file
+ <p style="text-align: center;">1</p>                         | To filter ImagePullSecrets, type characters in this field and then press the Enter key. All ImagePullSecrets that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                                                   |
```

Comment:
The preview of the list shows there are errors. Please check

![image](https://user-images.githubusercontent.com/43203744/105329830-8d01cb00-5c0c-11eb-8072-2d86c8147a7d.png)